index.html

```diff
@@ -69,8 +69,8 @@
                 <!-- Projects -->
                 <div class="collapse navbar-collapse" id="navbar-collapse">
                     <ul class="nav navbar-nav">                
-                        <li><a href="blocklyc.html?openFile=true">Open project</a></li>
-                        <li><a href="blocklyc.html?newProject=true">New project</a></li>
+                        <li><a id="open-project-link" class="editor-link" href="blocklyc.html?openFile=true">Open project</a></li>
+                        <li><a id="new-project-link" class="editor-link" href="blocklyc.html?newProject=true">New project</a></li>
                         <li id="noScript">
```

Comment:
Adds id's and classes so these can be accessed by the javascript.

Reply:
Are these new anchor id tags referenced somewhere else in the PR? If they are not, I do not see why adding the ids is necessary.

Reply:
I added them because my first attempt at this used an different approach - however I didn't delete them because if we do some UI work to the front page (#135) we will likely need them.

If you object, let me know and I'll delete them.

Reply:
AFAIK, the home page is going away at some point and the Solo will become a single page app. I would revert the additions until such time that we actually need them.

Reply:
Done.  IDs are gone, classes are used to pass parameters.